### ISSUE TITLE
Nose detection, eye detection

### ISSUE THREAD

**Issue by jeelnathani** [geekquad/Image-Processing-OpenCV#78]
Hello, I am GSSOC '21 participant. Just like face detection, I would like implement nose and eye detection using opencv

**Comment by geekquad** [comment]
Hey @jeelnathani, isn't it a part of face detection?

**Comment by jeelnathani** [comment]
@geekquad yes. for nose and eye detection, first we have to detect face. when I was doing these detections, detecting eye was easy but it was hard to find xml file for nose detection (most of the file posses error). so, I would like to contribute here

**Comment by geekquad** [comment]
Hey @jeelnathani, would like to let you know that we are majorly focusing on Image Processing techniques here. As far as I know, this is majorly an application-based method. 
However, you can still go ahead with this. Focus more on the implementation of the core concept rather than its application. Make a PR and we will see.
